### DISCUSSION TITLE
Problems with sublist and code

### DISCUSSION THREAD

**Discussion by filirnd** [mkdocs/mkdocs#3416] [Q&A]
Hi, 
I have a sublist with code like **this**:
____________________________________________________________
3. Attraverso **docker-compose**: 
    - Con il comando make create è possibile creare il docker del microservizio; 
    - Una volta creato, è possibile utilizzare questo esempio di docker-compose:<br>
   ```
    version: '3.8' 
    services: 
   ```

    - La parte volumes è opzionale e serve ad utilizzare una eventuale configurazione esterna che va a sostituire quella presente dentro il docker..
____________________________________________________________

But it is rendered with problems:
![issuemkdocs](https://github.com/mkdocs/mkdocs/assets/6331221/e4e44604-068d-4526-96e4-92dabf2136c5)

How can I resolve this?

On **markdown online tools** it works great 

![image](https://github.com/mkdocs/mkdocs/assets/6331221/7f329b6a-cde4-4741-b844-ceb9782e1429)

mkdocs  **version 1.5.3** 
theme  **mkdocs-material Version: 9.4.3**

thank you in advance

**Comment by pawamoy** [comment]
Hello, it's working fine on my end:

![mkdocs3415](https://github.com/mkdocs/mkdocs/assets/3999221/51baf5d3-b3d5-4e57-aac8-0725744a25ad)

Can you show your configured Markdown extensions?

**Reply by filirnd** [reply]
This is my config

site_name: Developer Guide
theme:
  name: material
  features:
    - content.code.copy
markdown_extensions:
    codehilite:
        use_pygments: False
plugins:
  - search
  - i18n:
      docs_structure: folder
      languages:
        - locale: en
          default: true
          name: English
          build: true
        - locale: it
          name: Italian
          build: true
